dasgo/requests.go

```diff
@@ -1086,6 +1088,8 @@ type GetGuildVanityURL struct {
 	Uses    int    `json:"uses,omitempty"`
 }
 
+// TODO: no table
```

Comment:
Table is provided but as URL Query strings which are included in the url hence the `url` tag. This one is correct.